src/views/shared/FooterHideableBackLink.js

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define(['./FooterSignout', 'okta', 'util/Enums'], function (FooterSignOut, Okta, Enums) {
+  let { Util } = Okta.internal.util;
+  let _ = Okta._;
+
+  return FooterSignOut.extend({
```

Comment:
Naming is a bit misleading, it extends `FooterSignOut` but file name is `FooterHideableBackLink.js`. 
- So what's exact purpose ?
- also looks like you'd rather update the template in parent like what you have done for `ForgotPasswordController.js`?

Reply:
It's been updated.